network/manager.go

```diff
@@ -163,7 +162,7 @@ func (nm *networkManager) restore(isRehydrationRequired bool) error {
 					for _, extIf := range nm.ExternalInterfaces {
 						for _, nw := range extIf.Networks {
 							log.Printf("[net] Deleting the network %s on reboot\n", nw.Id)
-							nm.deleteNetwork(nw.Id, &cni.NetworkConfig{})
+							nm.deleteNetwork(nw.Id, &cni.NetworkConfig{})  //nolint:errcheck // error is not needed to be checked in rehydration
```

Comment:
you should discard the error explicitly like `_ = rm.deleteNetwork` so that it's clear to future readers that this was intentional and not a mistake